tokio-macros/src/entry.rs

```diff
@@ -437,7 +531,7 @@ fn parse_knobs(mut input: ItemFn, is_test: bool, config: FinalConfig) -> TokenSt
         quote! {}
     };
 
-    let body_ident = quote! { body };
+    let body_var = Ident::new("body", Span::call_site());
     // This explicit `return` is intentional. See tokio-rs/tokio#4636
     let last_block = quote_spanned! {last_stmt_end_span=>
```

Comment:
Just saw this PR, given my attempts at https://github.com/tokio-rs/tokio/pull/6882 I just want to raise that these `quote_spanned! {last_stmt_end_span`s are also confusing rust-analyzer a fair bit when it comes to completions. That was my main motivation when I authored https://github.com/tokio-rs/tokio/pull/6882.